rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides analysis that may require individual attention when
+            migrating to newer Hibernate java constructs.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate" versionRange="[,3.9)" />
+        <targetTechnology id="hibernate" versionRange="[4,)" />
+        <tag>reviewed-2015-09-23</tag>
+    </metadata>
+
+    <rules>
+        <rule id="hibernate-catchall-00000">
+            <when>
+                <javaclass references="{package}.{type}" />
+            </when>
+            <perform>
+                <iteration>
+                    <when>
+                        <not>
+                            <has-hint />
+                        </not>
+                    </when>
+                    <perform>
+                        <classification title="Hibernate" severity="mandatory" />
+                        <hint effort="0" severity="mandatory" title="Hibernate type reference {package}.{type}">
+                            <message>
+                                This is a Hibernate type and needs to be migrated to a compatible API. There is currently no detailed
```

Comment:
I think this wording should be changed, as we (obviously) do have extensive documentation on hibernate itself.

Maybe something like:

"This is a Hibernate type and will need to be verified for compatibility with the latest version"